src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -36,7 +41,7 @@ const styles = StyleSheet.create({
     borderRadius: 16
   },
   anonImageBg: {
-    backgroundColor: COLORS.holyTosca
+    backgroundColor: colors.anon_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color property in the style for `anonImageBg` has been updated from `COLORS.holyTosca` to `colors.anon_primary`. This change is consistent with the PR description and there are no issues with this change.
```diff
-    backgroundColor: COLORS.holyTosca
+    backgroundColor: colors.anon_primary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->